### PR TITLE
Added sites

### PR DESCRIPTION
**Pull request by Jaki-py**

### CODE REVIEW

**Comment by Jaki-py** [comment]
Updated lists from pages in issue #133

**Review by Jhonsshons — APPROVED** [review]

File: lists/Urls/v.txt [hunk 1]
```diff
@@ -573,10 +580,4 @@ vtube.mobi
 vuku.ru
 vulgarasianphotos.com
 vulgarmoms.com
-vulgarsluts.com
```
Comment (mrvivacious) [inline lists/Urls/v.txt:576]: hi, why did u delete these? are they non-porn sites (i havent checked)

File: lists/Urls/t.txt [hunk 2]
```diff
@@ -1820,10 +1823,4 @@ twohornyguys.com
 twoplustwo.com
 twotgirls.com
 twpornstars.com
-twrank.com
```
Comment (mrvivacious) [inline lists/Urls/t.txt:1823]: why did u delete these?

**Comment by mrvivacious** [comment]
aweseome. please see my comments on the list deletion. after that, ill merge this.

good job getting familiar with adding stuff to the lists √

**Pull request closed without merging**